docs/Displaying a User Interface/Text Images and Buttons/index.md

```diff
@@ -325,6 +325,52 @@ retrievedSoftButtonObject.transitionToNextState();
 ```
 !@
 
+### Highlighting the Soft Button
```

Comment:
Move this section above `Updating the Soft Button State` to keep updating and deleting together. Highlighting doesn't really make sense right in the middle of them.

Reply:
moved